### PR TITLE
Fixup package releasing

### PR DESCRIPTION
**Pull request by gnunicorn**
fixes so unleash can release again.

- [x] additional metadata-field for newly added `sc-consensus`
- [x] fix future's [compat problem with sc-consensus-babe-rpc](https://gitlab.parity.io/parity/substrate/-/jobs/485874#L12039)
- [x] and [in other rpc-systems](https://gitlab.parity.io/parity/substrate/-/jobs/486594)...

waiting for unleash to confirm, then will revert.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -228,9 +228,9 @@ test-dependency-rules:
 unleash-check:
   stage:                           test
   <<:                              *docker-env
-  only:
-    - master
-    - tags
+  # only:
```
Comment (NikVolf) [inline .gitlab-ci.yml:231]: How is this related to PR?

File: client/consensus/common/src/lib.rs [hunk 2]
```diff
@@ -13,7 +13,8 @@
 
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
-//! Collection of consensus specific imlementations
+
+//! Collection of common consensus specific imlementations
```
Comment (NikVolf) [inline client/consensus/common/src/lib.rs:17]: And this?

File: .gitlab-ci.yml (hunk 1 above)
Comment (gnunicorn) [reply .gitlab-ci.yml:231]: it isn't. It is here to trigger the specific CI job to fix it. Once the CI passed, this is reverted.

File: client/consensus/common/src/lib.rs (hunk 2 above)
Comment (gnunicorn) [reply client/consensus/common/src/lib.rs:17]: usually just copy that first line into the field for crates (until we have some decent copyedits), and fixed it while I was at it.

Comment (bkchr) [inline client/consensus/common/src/lib.rs:17]:
```suggestion
//! Collection of common consensus specific implementations
```

**Comment by gnunicorn** [comment]
looks like a few more crates miss the `features = ["compat"]` in their dependencies...
